data/raw/keybindings.json

```diff
@@ -807,6 +807,12 @@
       { "input_method": "gamepad", "key": "JOY_3" }
     ]
   },
+  {
+    "type": "keybinding",
+    "id": "CHOOSE_DESTINATION",
+    "name": "Choose destination",
+    "bindings": [ { "input_method": "keyboard", "key": "F1" } ]
```

Comment:
I may be wrong, but I seem to recall issues with the function keys on some platforms